.env.template

```
@@ -224,6 +224,12 @@ LITELLM_LOG="ERROR"
 #AWS_SECRET_ACCESS_KEY=""
 #AWS_SESSION_TOKEN=""
 
+################################################################################
+#  WEB SCRAPER Configuarations
+################################################################################
+
+WEB_SCRAPER_TIMEOUT=15.0
+WEB_SCRAPER_MAX_DELAY=10.0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WEB_SCRAPER_MAX_DELAY key should go before the WEB_SCRAPER_TIMEOUT key

(UnorderedKey)